contributors/design-proposals/scheduling/cluster-scoped-resources.md

```diff
@@ -0,0 +1,76 @@
+# Cluster Scoped Resources
```

Comment:
please follow the KEP process outlined by @kubernetes/sig-architecture-feature-requests

Reply:
Is KEP now a requirement or a recommendation? That was not clear from the contributor summit discussions.

Reply:
/cc @jdumars

Reply:
@vishh @timothysc: is this the template that needs to be followed: https://github.com/kubernetes/community/blob/master/keps/0000-kep-template.md